sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -2489,9 +2499,26 @@ void ForUnderlyingType(TypedefDecl typedefDecl, Type underlyingType)
                 {
                     ForPointeeType(typedefDecl, parentType: null, referenceType.PointeeType);
                 }
-                else if (underlyingType is TagType)
+                else if (underlyingType is TagType underlyingTagType)
                 {
-                    // Nothing to do for tag types
+                    // See if there's a potential typedef remapping we want to log
```

Comment:
nit: Just for consistency, I think this logic should be in a `ForTagType` local function. It can then early exit with `if (!_config.LogPotentialTypedefRemappings)`